tools/headerparser/headerlib/struct_walker.py

```diff
@@ -0,0 +1,263 @@
+#
```

Comment:
Hi, "headerlib" is part of this tool, it's not external/third party, it was written from scratch for headerparser.